src/Tools/dotnet-monitor/Egress/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -129,6 +142,74 @@ private Uri GetAccountUri(AzureBlobEgressProviderOptions options, out string acc
             return blobUriBuilder.ToUri();
         }
 
+        private Uri GetQueueAccountUri(AzureBlobEgressProviderOptions options, out string accountName)
+        {
+            var queueUriBuilder = new QueueUriBuilder(options.QueueAccountUri);
```

Comment:
This can be null here. We probably need to check both `QueueName` and `QueueAccountUri` before entering this. If one is null with the other being set we should probably print a warning and not send anything to a queue.